docs/2.3/user-manual.md

```diff
@@ -72,7 +72,8 @@ certificates with a TTL (time to live) of 23 hours.
 
 !!! tip "Tip": 
     It is recommended to always use `tsh login` before using any other `tsh` commands.
-    This allows you to omit `--proxy` flag in subsequent tsh commands.
+    This allows you to omit `--proxy` flag in subsequent tsh commands; for example 
```

Comment:
This allows users to omit `--proxy` flag

Reply:
I'd make two sentences:
This allows users to omit `--proxy` flag in subsequent tsh commands. For example, `tsh ssh user@host` will work.